geoportal/c2cgeoportal_geoportal/scaffolds/update/{{cookiecutter.project}}/geoportal/CONST_vars.yaml

```diff
@@ -1205,8 +1205,10 @@ vars:
     level: 10
 
   # What web page is authorized to use the API
-  authorized_referers:
-    - '{VISIBLE_WEB_PROTOCOL}://{VISIBLE_WEB_HOST}/'
+  authorized_referers: *allowed_hosts
+
+  # Host allowed in the OGC server clear cache
+  allowed_hosts: *allowed_hosts
```

Comment:
It's a little wreid having allowed_hosts for the OGC server clear cache at the root when other are in subkeys.
And it also a little wreid having anchor defined in authentication.allowed_hosts and resused in allowed_hosts.

Reply:
> It's a little wreid having allowed_hosts for the OGC server clear cache at the root when other are in subkeys.

I agree, should we put in an `admin` or `themes` section?

> And it also a little wreid having anchor defined in authentication.allowed_hosts and resused in allowed_hosts.

Except the order I do that to simplify the follows, but I think that I should remove it and add some documentation here: https://camptocamp.github.io/c2cgeoportal/master/integrator/security.html, do you agree?

Reply:
Yes seems a really good idea to add some documentation about this in security.html

As far as I remember, OGC server cache clear is in themes because it was more easy to use existing code (reload of capabilities cache) but only requested from admin backend in a thread.

So from a user point of view this is an admin functionality. So I would vote for `admin`.

Note that we might change the url to `/admin/...` and require permission `admin`.